tests/components/material.test.js

```diff
@@ -70,6 +72,56 @@ suite('material', function () {
     });
   });
 
+  suite('updateSchema', function () {
+    test('the schema is updated', function () {
```

Comment:
ideally test name is "updates the schema" so the suite + test names form a full sentence